magicbyirineu/Modules/CardList/Interactor/CardListInteractor.swift

```diff
@@ -246,4 +324,43 @@ class CardListInteractor: NSObject {
         
         return objects
     }
+    
+    func appendCards(_ cards:[Card], totalExpected:Int){
```

Comment:
Acho que deixou vazar esse método vazio. Se for de um delegate assinado talvez faça mais sentido separar os delegates.